### ISSUE TITLE
No Development Instructions

### ISSUE THREAD

**Issue by vkottler** [badgerloop-software/pod3_server#1]
Copied from [an identical issue in pod3_dashboard](https://github.com/badgerloop-software/pod3_dashboard/issues/3):

This repository needs a `README.md` which allows one to write documentation leveraging [GitHub markdown](https://guides.github.com/features/mastering-markdown/).

For instance, what is this repository for? If someone wanted to work on it, how do they set up their computer? What operating systems are supported and what software do they need?

See [pod3_gcc](https://github.com/badgerloop-software/pod3_gcc), [gcc-builds](https://github.com/madison-embedded/gcc-builds) or [BadgerloopControls](https://github.com/vkottler/badgerloopControls) for examples.

**Comment by PaytonGarland** [comment]
Yep!  I actually began one in my branch which I will make a PR for soon.  I planned on mimicking the development instructions from one of the other projects, but just haven't gotten to writing it out yet.  Will be done soon.
